### PR TITLE
Add npm scripts to generate npm components package

### PR DESCRIPTION
**Pull request by t0oF**
The package can be generated only with npm (no python script).
It has been used by an npm project.

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -26,7 +27,9 @@
         "storybook": "start-storybook -p 6006",
         "build-storybook": "build-storybook",
         "test": "jest --coverage",
-        "test:watch": "npm test -- --watch"
+        "test:watch": "npm test -- --watch",
+        "build:npm": "rimraf dist && tsc --declaration && npm run copy-files",
```
Comment (t0oF) [inline react/package.json:31]: build:npm and copy-files scripts are inspired from webviz-core-components

File: react/src/lib/components/WellCompletions/components/WellCompletionComponent.tsx [hunk 2]
```diff
@@ -7,7 +7,8 @@ import DataProvider from "./DataLoader";
 import WellCompletionsViewer from "./WellCompletionsViewer";
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const inputSchema = require("../../../../../inputSchema/wellCompletions.json");
+const inputSchema = require("../../../inputSchema/wellCompletions.json");
```
Comment (t0oF) [inline react/src/lib/components/WellCompletions/components/WellCompletionComponent.tsx:10]: I couldn't find any other way to access this file both from webpack and when using the package as a library package.
When building an npm package, we want to include only the library part. Thus, inputSchema has to be at that level.

Note:
* in an ideal world, the demo and the library part should be in separate packages
* if there is a better solution I'll be happy to apply it, but I couldn't find any

File: react/tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
     "compilerOptions": {
         "outDir": "./dist/", // path to output directory
+        "declaration": true,
```
Comment (t0oF) [inline react/tsconfig.json:4]: This could actually be removed: I added the flag in the npm script

**Comment by t0oF** [comment]
Proposal to fix #466

**Review by anders-kiaer — APPROVED** [review]

**Pull request merged**